gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -157,38 +354,49 @@ public void start() throws IOException, InterruptedException {
         zipIn.closeEntry();
         entry = zipIn.getNextEntry();
       }
-    }
+    }   
+  }
+  
+  private void startGcd(Path executablePath) throws IOException, InterruptedException {
     // cleanup any possible data for the same project
-    File datasetFolder = new File(gcdFolder, GCD + '/' + projectId);
+    File datasetFolder = new File(gcdPath.toFile(), projectId);
     deleteRecurse(datasetFolder.toPath());
-
-    // create the datastore for the project
-    ProcessBuilder processBuilder = new ProcessBuilder()
-        .redirectError(ProcessBuilder.Redirect.INHERIT)
-        .directory(new File(gcdFolder, GCD));
+    
+    // create command wrappers
+    CommandWrapper gcdCreateCommand;
+    CommandWrapper gcdStartCommand;
+    Path gcdAbsolutePath;
     if (isWindows()) {
-      processBuilder.command("cmd", "/C", "gcd.cmd", "create", "-p", projectId, projectId);
-      processBuilder.redirectOutput(new File("NULL:"));
+      gcdCreateCommand = CommandWrapper.cmd();
+      gcdStartCommand = CommandWrapper.cmd();
+      gcdAbsolutePath = executablePath.toAbsolutePath().resolve("gcd.cmd");
     } else {
-      processBuilder.redirectOutput(new File("/dev/null"));
-      processBuilder.command("bash", "gcd.sh", "create", "-p", projectId, projectId);
+      gcdCreateCommand = CommandWrapper.bash();
+      gcdStartCommand = CommandWrapper.bash();
+      gcdAbsolutePath = executablePath.toAbsolutePath().resolve("gcd.sh");
     }
 
-    Process temp = processBuilder.start();
+    // create the datastore for the project
+    if (log.isLoggable(Level.FINE)) {
+      log.log(Level.FINE, "Creating datastore for the project: {0}", projectId);
+    }
+    Process temp = gcdCreateCommand.command(gcdAbsolutePath.toString(), "create", "-p", projectId,
```

Comment:
The preferred wrapping would be:
```java
    Process temp =
        gcdCreateCommand.command(gcdAbsolutePath.toString(), "create", "-p", projectId, projectId)
            .redirectErrorInherit()
            .directory(gcdPath)
            .redirectOutputToNull()
            .start();
```

see: http://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break
and https://github.com/google/styleguide for formatters.